plugins/push/frontend/public/templates/mobile-message-preview.html

```diff
@@ -0,0 +1,58 @@
+<div class="mobile-preview">
+    <div class="mobile-preview__images">
+        <img v-if="isIOSPlatformSelected()" src="/images/push/preview.i.png" />
+        <img v-else-if="isAndroidPlatformSelected()" src="/images/push/preview.a.png" />
+    </div>
+
+    <div v-if="isIOSPlatformSelected()" class="mobile-preview__ios-message">
+        <div class="mobile-preview__ios-message__header">
+            <img class="mobile-preview__ios-message__header__image" src="/appimages/5ea4bd674bb4a33298a81727.png" />
+            <span class="mobile-preview__ios-message__header__app-name">
+                {{appName}}
+            </span>
+            <span class="mobile-preview__ios-message__header__close-button">
+                X
+            </span>
+        </div>
+        <img v-if="media" class="mobile-preview__ios-message__media" v-bind:src="media" /> 
+        <div class="mobile-preview__ios-message__title">
+            {{title}}
+        </div>
+        <div class="mobile-preview__ios-message__content">
+            {{content}}
+        </div>
+        <div class="mobile-preview__ios-message__buttons-list">
+            <div v-for="button in buttons" class="mobile-preview__ios-message__buttons-list__item">{{button}}</div>
```

Comment:
i think we had a small discussion about nesting in BEM.
we should try to avoid it.
https://en.bem.info/methodology/quick-start/#guidelines-for-using-elements